frontend/src/assets/css/styles.scss

```diff
@@ -68,3 +68,26 @@ code {
     transform: translateY(0px)
   }
 }
+
+// login
+.auth-container {
+  width: 100%;
```

Comment:
You can use Bootstrap classes for this. w-100, h-100.

Please check all styles and use bootstrap classes instead of custom styles.